README.md

```diff
@@ -56,8 +56,19 @@ AWS_CRT_BUILD_USE_SYSTEM_LIBCRYPTO=1 python3 -m pip install --no-binary :all: --
 ```
 ( `--no-binary :all:` ensures you do not use the precompiled wheel from PyPI)
 
-You can ignore all this on Windows and Apple platforms, where aws-crt-python
-uses the OS's default libraries for TLS and cryptography math.
+In addition, aws-crt-python also exposes a number of cryptographic primitives. 
+On Unix, those depend on libcrypto as described above.
+On Apple and Windows OS level crypto libraries are used whenever possible.
+One exception to above statement is that for ED25519 keygen on Windows and Apple, 
+libcrypto is used as no viable OS level alternative exists. In that case Unix level notes
+about libcrypto apply to Apple and Windows as well. Libcrypto usage for ED25519 support is 
+enabled on Windows and Apple by default and can be disabled by setting environment variable
+`AWS_CRT_BUILD_DISABLE_LIBCRYPTO_USE_FOR_ED25519_EVERYWHERE` as follows:
```

Comment:
ignore me: some day, we'll use it for some other operations too, and we'll need to rename this, but it's so niche that I'm not going to get all worked up about it